app-emulation/docker-credential-helpers/metadata.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+    <longdescription>
+		A suite of programs to use native stores to keep Docker credentials
+		safe.  Currently provides docker-credential-secretservice to use the
+		D-Bus secret service APIs, and docker-credential-pass to use the
+		pass framework.
+    </longdescription>
+	<maintainer type="person">
+		<email>rkitover@gmail.com</email>
+		<name>Rafael Kitover</name>
+	</maintainer>
+	<maintainer type="project">
+		<email>proxy-maint@gentoo.org</email>
+		<name>Proxy Maintainers</name>
+	</maintainer>
+	<maintainer type="project">
```

Comment:
Don't add other projects without their permission.

Reply:
Asking in `#gentoo-containers` now, if no one responds I will remove it.

Reply:
Well I got no response so I removed this block.
BTW I really appreciate your help and your patience.